FrontEnd/src/components/ProfilePage/FormComponents/GeneralInfo.js

```diff
@@ -50,6 +52,8 @@ const ERRORS = {
 const TEXT_AREA_MAX_LENGTH = 2000;
 const BANNER_IMAGE_SIZE = 5 * 1024 * 1024;
 const LOGO_IMAGE_SIZE = 1 * 1024 * 1024;
+const RNOKPP_PATTRN = /^\d{10}$/;
+const EDRPOU_PATTERN = /^\d{8}$/;
```

Comment:
Shouldn't the patterns be placed in utils module or inside the validation functions ecpeccially when those functions are specific for given values?